tests/pd/test_rpc_client.rs

```diff
@@ -113,27 +114,50 @@ fn test_validate_endpoints() {
 }
 
 #[test]
-fn test_change_leader() {
+fn test_retry_async() {
```

Comment:
See line https://github.com/pingcap/tikv/pull/1712/files/ceb73833339cbe3dfcd4b46e5a7237b79303fd6c#diff-734743149ab1e2c4cf8617bf000970ccR48, the mock server may returns an `Err(_)`, if the client don't retry it panics at https://github.com/pingcap/tikv/pull/1712/files/ceb73833339cbe3dfcd4b46e5a7237b79303fd6c#diff-307a5e1916813c7a90d5fd89f1cbc5ebR133